packages/editor/src/components/header/back-button.js

```diff
@@ -9,16 +9,16 @@ import {
 // Keeping an old name for backward compatibility.
 const slotName = '__experimentalMainDashboardButton';
 
+export const useHasBackButton = () => {
+	const fills = useSlotFills( slotName );
+	return Boolean( fills && fills.length );
+};
+
 const { Fill, Slot } = createSlotFill( slotName );
 
 const BackButton = Fill;
-const BackButtonSlot = ( { children } ) => {
+const BackButtonSlot = () => {
 	const fills = useSlotFills( slotName );
-	const hasFills = Boolean( fills && fills.length );
-
-	if ( ! hasFills ) {
-		return children;
-	}
```

Comment:
@stokesman Is this change required? If yes, can you elaborate more on "why"? Thanks.

Reply:
I didn’t have to remove this but it seemed redundant if we let the `Header` decide whether to render the slot. "Why" is hopefully clarified in my reply to your other comment on the header.

Reply:
Thanks for the explanation, @stokesman! I guess it's okay to change the component's behavior; it's a private component, so there are no requirements for BC.